pkgs/lightning-pool/get-sha256.sh

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -i bash -p git gnupg
+set -euo pipefail
+
+TMPDIR="$(mktemp -d -p /tmp)"
+trap "rm -rf $TMPDIR" EXIT
+cd $TMPDIR
+
+echo "Fetching latest release"
+git clone https://github.com/lightninglabs/pool 2> /dev/null
+cd pool
+latest=v0.4.3-alpha
```

Comment:
Is there a reason why you're not using the "latest release" logic from the other `get-sha256` scripts?

Reply:
I think lightning labs did something wrong with their tagging. If i use the other logic i get the [auctioneerrpc/v1.0.1](https://github.com/lightninglabs/pool/tree/auctioneerrpc/v1.0.1) tag

Reply:
The failure results from multiple tags being on the same commit. Let's use the default logic on the next tag, which will hopefully be the only one on that commit.